src/ring_theory/ideal/operations.lean

```diff
@@ -172,18 +167,15 @@ smul_le.2 $ λ s hs n hn, show r • (s • n) ∈ (I • J) • N,
 lemma smul_inf_le (M₁ M₂ : submodule R M) : I • (M₁ ⊓ M₂) ≤ I • M₁ ⊓ I • M₂ :=
 le_inf (submodule.smul_mono_right inf_le_left) (submodule.smul_mono_right inf_le_right)
 
-lemma smul_supr {ι : Sort*} {I : ideal R} {t : ι → submodule R M} :
-  I • supr t = ⨆ i, I • t i :=
+lemma smul_supr {ι : Sort*} {I : ideal R} {t : ι → submodule R M} : I • supr t = ⨆ i, I • t i :=
```

Comment:
Same here, the arguments take up more visual space than the return type but are much less relevant.
```suggestion
lemma smul_supr {ι : Sort*} {I : ideal R} {t : ι → submodule R M} :
  I • supr t = ⨆ i, I • t i :=
```